src/systemctl/systemctl.c

```diff
@@ -926,7 +926,7 @@ static int systemctl_parse_argv(int argc, char *argv[]) {
         return 1;
 }
 
-static int parse_argv(int argc, char *argv[]) {
+int systemctl_dispatch_parse_argv(int argc, char *argv[]) {
```

Comment:
this rename isn't really necessary, is it?

Reply:
I'd have to put `parse_argv` in a header, which seemed like an invitation for an issue later on. I think a fully generic name like `parse_argv` should always stay `static`.